show_notes/Episode_18_Part1.md

```diff
@@ -0,0 +1,18 @@
+# Episode 18, Part 1: Android Everywhere
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+* [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/92)
+
+In the first part of episode 18 Artur, Hannes und Artem discuss about android tablets, Chromebooks and Wear OS by Google.
```

Comment:
In the first part of the episode #18 Artur, Hannes and Artem discuss Android tablets, Chromebooks and Android Wear (Wear OS).